### ISSUE TITLE
Show Date in All History

### ISSUE THREAD

**Issue by clayop** [cryptonomex/graphene-ui#580]
According to wmbutler's screenshot, date section in all history shows both date and time. Now the GUI only presents time.

![image](https://cloud.githubusercontent.com/assets/9971632/11578343/7be24656-99d9-11e5-8d0b-6d1acd9c6820.png)

**Comment by wmbutler** [comment]
Yeah, I'd really prefer that but @svk31 had concerns about spacing. Can we force the min width to be wide enough to support the date as well?

This will likely force users with small screens to view the bid/ask numbers below the chart instead of beside the chart but I think it's a fine compromise....also:

Can we fix the width of the right column (history and markets). I'd prefer that it doesn't flex. Let's let the market flex.

TLDR;
- [x] Add date to history as shown
- [x] Fixed width of the right column (market and history)
